src/content/docs/validator/external-chains/xrpl.mdx

```diff
@@ -145,15 +145,14 @@ curl -H 'Content-Type: application/json' -d '{"method":"server_info","params":[{
 
 
 ### Ampd Configuration
-Once your node is up and running, you need to add the XRPL chain to your ampd's `config.toml`file. This can be done by adding in the following configuration:
+Once your node is up and running, you need to add the `XRPL` chain to your [ampd daemon's](https://github.com/axelarnetwork/axelar-amplifier/tree/main/ampd) `config.toml`file. This can be done by adding in the following configuration:
 
 For testnet:
-
 ```toml
 [[handlers]]
 type="XRPLMsgVerifier"
 chain_name="xrpl"
-chain_rpc_url=<YOUR_RPC_URL>
+chain_rpc_url="your_own_rpc"
 cosmwasm_contract="axelar1pnynr6wnmchutkv6490mdqqxkz54fnrtmq8krqhvglhsqhmu7wzsnc86sy"
```

Comment:
needs a verifier set verifier

Reply:
actually sorry, I was mistaken, there's no verifier set handler for xrpl. That functionality is just supported by the message verifier handler